job-descriptions/software-engineer-backend.md

```diff
@@ -0,0 +1,61 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Backend
+
+### About us
+
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
+
+We value openness and transparency. [Our code](https://github.com/sourcegraph/sourcegraph), [our roadmap](https://about.sourcegraph.com/direction), and [our company processes](https://about.sourcegraph.com/company/open_source_open_company) are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+
+To create a product that serves the needs of all developers, we are building a diverse remote-first team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+As a backend engineer at Sourcegraph, you will focus on designing and developing fast and reliable services that power our product (e.g., code search, code intelligence, automation). Read our [project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.g2wq4qci7wj0) to see the kinds of projects that we are working on today and have planned for the future.
+
+We are looking for candidates with the following skills and a strong desire to learn:
+
+- Excellent knowledge of Go (preferred) or TypeScript.
+- Skilled at building and testing (e.g., unit testing, automated end-to-end testing) high performance distributed systems in Go (preferred) or TypeScript.
+- Ability to communicate clearly and empathetically, especially in writing.
+- Practiced at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g., cut scope) to ship quickly and iterate when necessary.
+
+Nice to have:
```

Comment:
Either we move the nice to haves I proposed above here or we merge the two sections and tag each nice to have in-line.

Reply:
I think explicitly separating them into two lists and moving your nice-to-haves down here is better since it makes the "required" list look smaller and thus reduces possible hesitation to apply but still gives a good description of what we're looking for.

Reply:
Structurally I think it is important and valuable to separate requirements from nice-to-have.